multiple pages: Add pt_PT translations (#3540)
tldr-pages/tldr: pages.pt_PT/common/touch.md

# touch

> > Atualizar as timestamps dos ficheiro para a hora atual.

Reviewer: ```suggestion
> Atualizar as timestamps do ficheiro para a hora atual.
```

Reviewer 2: Eu preferiria uma formulação mais genérica:

`Atualizar as timestamps de ficheiros para a hora atual.`

> > Se o ficheiro não existir, cria ficheiro vazio, a menos que seja passado parâmetro -c ou -h.

Reviewer 2: ```suggestion
> Se o ficheiro não existir, cria um ficheiro vazio, a menos que seja passado o parâmetro -c ou -h.
```

> - Cria novo(s) ficheiro(s) vazio ou atualiza as timestamps para a hora atual:

Reviewer 2: Deve-se usar o infinitivo nas descrições:

```suggestion
- Criar um novo ficheiro vazio, ou atualizar as timestamps para a hora atual:
```

`touch {{ficheiro}}`

> - Define as timestamps de um ficheiro para a hora especificada:

Reviewer 2: É preciso mudar em todos os exemplos de todas as páginas para usar o infinitivo.

Reviewer 2: Submitted as part of #4235.

> `touch -t {{YYYYMMDDHHMM.SS}} {{nome_ficheiro}}`

Reviewer 2: Por consistência com os outros exemplos:
```suggestion
`touch -t {{YYYYMMDDHHMM.SS}} {{ficheiro}}`
```

> - Usa as timestamps do ficheiro1 para definir as horas do ficheiro2:

Reviewer 2: Por consistência:
```suggestion
- Usa as timestamps do ficheiro1 para definir as timestamps do ficheiro2:
```
